list

```diff
@@ -389,6 +389,9 @@ gq.mn
 gr.pn
 grb.to
 grhb.me
+gtly.ink
+gtly.io
```

Comment:
I can't find a working example of `gtly.io`, @geotargetly would you like to provide some?

Reply:
We use gtly.io as an intermediate URL for custom short url domain mapping. For example www.medi.biz/synergie will route to 2d4e2d6273366735574b71746e75303973625372.gtly.io/synergie which will then route to final destination URLs.

Reply:
Hi @geotargetly, thanks for the update, in that case, I'd suggest to remove `gtly.io`, though some filters support to block/allow subdomains, the intention here for the list is to list those domains exactly been using as a URL shortener/redirecter, so the other two domains look good here ;)

Reply:
Totally understand :) let's keep gtly.io out of this list. Thanks for your help!